blocks/library/block/index.js

```diff
@@ -30,6 +30,7 @@ class ReusableBlockEdit extends Component {
 		this.state = {
 			isEditing: !! ( reusableBlock && reusableBlock.isTemporary ),
 			title: null,
+			attributes: null,
```

Comment:
I think the name of the state property could be improved to better indicate that it's the _transient_ updates which have yet to be saved. Otherwise it's a bit difficult to discern the difference between `this.state.attributes` and `this.props.block.attributes`.

Reply:
👌 fixed in 5a36e93ce.